uvcombine/tests/test_feather.py

```diff
@@ -90,9 +114,13 @@ def test_feather_simple_mismatchunit(plaw_test_data):
         combo = feather_simple(highres_hdu, lowres_hdu, match_units=False)
 
 
-def test_fourier_combine_cubes(plaw_test_cube_sc):
+def test_fourier_combine_cubes(cube_data):
 
-    orig_cube, sd_cube, interf_cube = plaw_test_cube_sc
+    orig_fname, sd_fname, interf_fname = cube_data
+
+    orig_cube, orig_data = cube_and_raw(orig_fname, use_dask=False)
+    sd_cube, sd_data = cube_and_raw(sd_fname, use_dask=False)
+    interf_cube, interf_data = cube_and_raw(interf_fname, use_dask=False)
```

Comment:
any reason `use_dask` isn't looped over here?

Reply:
Uses the old cube feather function that doesn't have dask handling.